python/GafferSceneUI/LightEditor.py

```diff
@@ -319,6 +320,57 @@ def __buttonDoubleClick( self, pathListing, event ) :
 
 		return True
 
+	def __keyPress( self, pathListing, event ) :
+
+		if event.key == "Return" or event.key == "Enter" :
+
+			selection = pathListing.getSelection()
+			columns = pathListing.getColumns()
+
+			inspections = []
+
+			with Gaffer.Context( self.getContext() ) as context :
+				for i in range( 0, len( columns ) ) :
+					column = columns[ i ]
+					if not isinstance( column, _GafferSceneUI._LightEditorInspectorColumn ) :
+						continue
+					for path in selection[i].paths() :
+						context["scene:path"] = GafferScene.ScenePlug.stringToPath( path )
+						inspection = column.inspector().inspect()
+
+						if inspection is not None :
+							inspections.append( inspection )
+
+			nonEditable = [ i for i in inspections if not i.editable() ]
+
+			if len( nonEditable ) == 0 :
+				edits = [ i.acquireEdit() for i in inspections ]
```

Comment:
The plugs acquired here can be of different types, in which case we get an exception like this :

```
File "/home/john/dev/build/gafferPython3/python/GafferSceneUI/TweakPlugValueWidget.py", line 82, in __init__
    self.__row.append( GafferUI.PlugValueWidget.create( self.__childPlugs( plugs, "value" ) ), expand = True )
  File "/home/john/dev/build/gafferPython3/python/GafferUI/PlugValueWidget.py", line 252, in create
    raise Exception( "Multiple widget creators" )
Exception: Multiple widget creators
```

In the Spreadsheet, we have a PopupWindow saying "Unable to edit plugs with mixed types". I think we need to do something similar here.

Reply:
It's perhaps not the best solution, but I added a dialog in 029cbf28f4dd2bbdbd080496029ffaa85503f21e warning of mismatched plugs.

I included a todo that I think might be an improvement that can be made on `0.62` moving the handling of `TweakPlug` with different value types into `PlugPopup` that might be a bit cleaner.